api/sendfild/pb/sendfil.proto

```diff
@@ -0,0 +1,103 @@
+syntax = "proto3";
+package api.sendfild.pb;
+option go_package = "github.com/textileio/textile/v2/api/sendfild/pb";
+
+import "google/protobuf/timestamp.proto";
+
+enum MessageState {
+  MESSAGE_STATE_UNSPECIFIED = 0;
+  MESSAGE_STATE_PENDING = 1;
+  MESSAGE_STATE_ACTIVE = 2;
+  MESSAGE_STATE_FAILED = 3;
+}
+
+enum WaitingFilter {
+  WAITING_FILTER_UNSPECIFIED = 0;
+  WAITING_FILTER_WAITING = 1;
+  WAITING_FILTER_NOT_WAITING = 2;
```

Comment:
It means the txn is pending AND the service actively has a `StateWaitMsg` call to louts running. The concept of "waiting" is really an internal thing, but I thought it would be a useful metric for knowing the health of the service, so I exposed it in the API.